virt/virt-4-8-release-notes.adoc

```diff
@@ -58,6 +58,7 @@ The SVVP Certification applies to:
 //CNV-9054 MAC address pool is now enabled on all namespaces by default.
 
 //CNV-9055 Kubernetes NMState now supports new IP configuration options.
+* You can use the Kubernetes NMSstate Operator to xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-example-nmstate-IP-management_virt-updating-node-network-config[configure and manage IP addresses] on your cluster node network.
```

Comment:
I would drop "network" from "node network". It seems redundant and slightly confusing - you don't manage the whole node network (e.g. you cannot control IP ranges of network HW such as routers). You can really only control IP addresses of nodes.

Reply:
Removed the word "network"